.eslintignore

```diff
@@ -0,0 +1 @@
+examples
```

Comment:
Why do we need to ignore this folder?

Reply:
the eslint at the root must not impact this folder